### ISSUE TITLE
devtool script should run in CI; required for PR approval

### ISSUE THREAD

**Issue by mkrufky** [livepeer/go-livepeer#919]
The devtool script is very helpful for development and testing.  We should always be sure that any changes to procedure are accounted for in this script.

**Comment by kyriediculous** [comment]
The only thing I'm not certain about is , should we run the generated  bash scripts in `devtool.go` as part of this additional check.
Considering flags and their respective arguments should already be covered by `test_args.sh`

**Comment by j0sh** [comment]
The tricky part here is that devtool expects a geth running somewhere with the Livepeer contracts deployed. If we can figure out how to get that working for CI then that'd be great.

**Comment by kyriediculous** [comment]
I believe @mkrufky ran this against a docker image Ivan made (#918) . We should be able to run the script against the image, no?

**Comment by yondonfu** [comment]
We should be able pull down the [geth-with-livepeer-protocol](https://github.com/livepeer/docker-livepeer/tree/master/geth-with-protocol) Docker image that is now hosted on the Livepeer DockerHub.

There is also an open [issue](https://github.com/livepeer/protocol/issues/295) to auto-build and push a new image whenever a new contracts PR lands on the `pm` branch.

**Comment by ericxtang** [comment]
@mk-livepeer can we close this issue now that the PR is merged?

**Comment by mkrufky** [comment]
No, the PR that was merged mentions this issue but does not close it.
